source/wp-content/themes/wporg-showcase-2022/src/site-meta-list/index.php

```diff
@@ -29,114 +45,74 @@ function render( $attributes, $content, $block ) {
 	$meta_fields = array(
 		array(
 			'label' => __( 'Author', 'wporg' ),
+			'type' => 'meta',
 			'key' => 'author',
 		),
 		array(
 			'label' => __( 'Country', 'wporg' ),
+			'type' => 'meta',
 			'key' => 'country',
 		),
 		array(
-			'label' => __( 'Theme', 'wporg' ),
-			'key' => 'theme',
+			'label' => __( 'Category', 'wporg' ),
```

Comment:
Should we limit the category to 1? 
<img width="373" alt="Screenshot 2023-08-09 at 9 46 11 AM" src="https://github.com/WordPress/wporg-showcase-2022/assets/1657336/2aef056b-b021-446b-9dee-cf96d12ac92b">

Reply:
Is the issue that you don't want to see Featured? We can omit that category, but I don't think this value needs to be limited to one.